lib/config/options/index.ts

```diff
@@ -1838,6 +1839,14 @@ const options: RenovateOptions[] = [
     cli: false,
     advancedUse: true,
   },
+  {
+    name: 'includeVulnerabilitySeverity',
```

Comment:
i would prefer to avoid this option and instead allow the user to override the template.

@rarkins WDYT?

Reply:
Yes, I'd prefer to make it a preset to opt into

Reply:
I originally tried overriding `commitMessageSuffix` using a packageRule (ie: without any `includeVulnerabilitySeverity` flag).

However from my local testing I wasn't able to override the suffix - it would always stay as `[SECURITY]`  🤷 

I imagine the same would be true if I moved this to an opt-in preset

Any tips?

Reply:
you probably need to move it to the global vulnerability alert config object, because tht is applied last